src/sentry/relay/config/__init__.py

```diff
@@ -49,7 +49,6 @@
 EXPOSABLE_FEATURES = [
     "projects:span-metrics-extraction",
     "organizations:transaction-name-mark-scrubbed-as-sanitized",
-    "organizations:transaction-name-normalize",
```

Comment:
I would keep this feature flag exposed, for older relay versions.

Reply:
@jjbayer  iker replied to you about this here: https://github.com/getsentry/sentry/pull/47080#discussion_r1165500908 whats your thoughts on his feedback ?

Reply:
Replied in old thread as to why I think its worth keeping this feature flag.